src/components/heading/FilterPrice.jsx

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react';
+
+import { Controller, useFormContext } from 'react-hook-form';
+
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import { TextField } from '@mui/material';
+
+function FilterPrice() {
+  const [open, setOpen] = useState(false);
+
+  const { control, setValue } = useFormContext();
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const minPrice = (price) => {
```

Comment:
Function naming should mirror action, in this case it should say something like `setMinPrice`